feat(scim): Change messaging for okta beta (#4042)
getsentry/sentry-docs: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx

### Requirements

Okta SCIM provisioning requires:

- A subscription to Sentry Business Plan or higher.
- Configuration of SAML SSO for Okta [as documented here](/product/accounts/sso/okta-sso/), or use the Okta sign-on tab in your Sentry Okta application to configure SAML.

> - **You must select Email for the Application username format** in the Sign On application tab in Okta.

Reviewer: Suggestion for parallel structure of the list

```suggestion
- **Selection of Email for the Application username format** in the Sign On application tab in Okta.
```
